packages/api/src/channels/channels.controller.ts

```diff
@@ -65,6 +65,19 @@ export class ChannelsController {
     }
   }
 
+  @Get()
+  @ApiOperation({ description: 'Retrieves the most recently verified 30 channels desc by date' })
+  @ApiResponse({ type: ChannelDto })
+  async getRecentVerifiedChannels() {
+    try {
+      const channels = await this.channelsService.getRecent(2)
```

Comment:
Description says 30 channels, but here you pass `2` as a limit

Reply:
Sorry, it's a debugging mistake. I will fix it